scripts/eurostat/regional_statistics_by_nuts/education_attainment/Eurostats_NUTS2_Edat.mcf

```diff
@@ -0,0 +1,129 @@
+# Population aged 25-64 by educational attainment level (0-2)
+Node: dcid:Count_Person_25To64Years_LessThanPrimaryEducationOrPrimaryEducationOrLowerSecondaryEducation_AsAFractionOfCount_Person_25To64Years
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+age: [Years 25 64]
+educationalAttainment: dcs:LessThanPrimaryEducation&PrimaryEducation&LowerSecondaryEducation
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+measurementDenominator: dcs:Count_Person_25To64Years
+
+# Population aged 25-64 by educational attainment level (3-8)
+Node: dcid:Count_Person_25To64Years_UpperSecondaryEducationOrHigher_AsAFractionOfCount_Person_25To64Years
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+age: [Years 25 64]
+educationalAttainment: dcs:UpperSecondaryEducationOrHigher
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+measurementDenominator: dcs:Count_Person_25To64Years
+
+# Population aged 25-64 by educational attainment level (3-4)
+Node: dcid:Count_Person_25To64Years_UpperSecondaryEducationOrPostSecondaryNonTertiaryEducation_AsAFractionOfCount_Person_25To64Years
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Person
+age: [Years 25 64]
+educationalAttainment: dcs:UpperSecondaryEducation&PostSecondaryNonTertiaryEducation
```

Comment:
Is the "&" correct?

Reply:
@jeffreyoldham Yes, this is how they decided to aggregate different enums
import doc: https://docs.google.com/document/d/192NoZd_-O5SfsHGkBiHteWbgjqCZkD8LUcSfAE_Icak/edit#